Bottom: 554e6b222e782e46b98a3cd6e93fd0556bdee98b
Top:    0d51a85de97c601609590dcc7326e6da10b941a8
Author: Ian Kent <raven@themaw.net>
Date:   2013-03-03 10:37:57 +0800

Refresh of autofs-5.0.7-dont-fail-on-master-map-self-include.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index b13a5f5..97d6f48 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -28,7 +28,7 @@
 - make yellow pages support optional.
 - modules/replicated.c: use sin6_addr.s6_addr32.
 - workaround missing GNU versionsort extension.
-- fix map read fail incorrectly set on master re-read.
+- dont fail on master map self include.
 
 25/07/2012 autofs-5.0.7
 =======================
